packages/scandipwa/src/component/MyAccountOrder/MyAccountOrder.style.scss

```diff
@@ -45,9 +45,15 @@
         margin-inline-end: 20px;
 
         &.Button {
-            &:hover {
+            &:active {
                 color: var(--link-hover);
             }
+
+            @media (hover: hover) and (pointer: fine) { 
```

Comment:
This is a beautiful solution, but too complex to memorize.
Please make a helper mixin for it and replace all occurrences with the new mixin.
As an example, you can see `@desktop` or `@mobile` mixins.